Doc/whatsnew/3.11.rst

```diff
@@ -624,6 +624,11 @@ enum
 
 * ``EnumMeta`` renamed to ``EnumType`` (``EnumMeta`` kept as alias).
 
+* The output of ``Enum.__format__`` has changed to include both the enum name
```

Comment:
Might be worth adding explicitly that this affects enums included in f-strings, since that may not be obvious to a lot of readers.